dts/bindings/input/linux-evdev.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright 2023 Google LLC
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  Linux evdev based input device
+
+  Allows using a Linux evdev device to read input events and report them back
+  as Zephyr input events.
+
+  Example configuration:
+
+  evdev {
+          compatible = "linux-evdev";
+  };
+
+  Then run the binary specifying the evdev device with the --evdev flag, for
+  example:
+
+  ./build/zephyr/zephyr.exe --evdev=/dev/input/event0
+
+compatible: "linux-evdev"
```

Comment:
Should this be `zephyr,linux-evdev`?

Reply:
uhm good point, should probably be `zephyr,native-linux-evdev` actually

Reply:
changed